coordinator/src/main/java/com/tencent/rss/coordinator/ApplicationManager.java

```diff
@@ -217,8 +223,53 @@ private void statusCheck() {
         appIdToRemoteStoragePath.remove(appId);
       }
       CoordinatorMetrics.gaugeRunningAppNum.set(appIds.size());
+      updateRemoteStorageMetrics();
     } catch (Exception e) {
       LOG.warn("Error happened in statusCheck", e);
     }
   }
+
+  private void updateRemoteStorageMetrics() {
+    for (String remoteStoragePath : availableRemoteStoragePath) {
```

Comment:
Do we need to update all paths?

Reply:
For coordinator, the metrics is collected in regular thread, and update all metrics according to current status.
Unnecessary update will be happened if there is no changed from last status check, but it shouldn't cost much resource. 
To simplify the implementation, I prefer to update all metrics with remote storage.

Reply:
OK